### ISSUE TITLE
[MU4 Issue] priority between a return to the line and MMrest

### ISSUE THREAD

**Issue by Gai-Luron** [musescore/MuseScore#14252]
**Describe the bug**
priority between a return to the line and MMrest. (Same in MS 3.6)

**To Reproduce**
1. Create 4 Mesure empty
![image](https://user-images.githubusercontent.com/1835871/199294221-a1bc7849-52ee-4d7c-b131-f4b1af0e3edf.png)
2. Add a CR at Measure 2, Two measure in a line, Tho measure in other line
![image](https://user-images.githubusercontent.com/1835871/199293720-7c3dc712-60d8-406c-b12d-da3d9f8d5866.png)
3. Go in MMrest mode, push on "M"
4. The four Measure are in the same line
![image](https://user-images.githubusercontent.com/1835871/199293836-98b8ee0d-a4a2-4c7c-97ff-a8899bdb8946.png)
5. expected
![image](https://user-images.githubusercontent.com/1835871/199293918-ad33926f-b199-4d23-ae00-7dbee9b94809.png)

**Platform information**
OS: Windows 10 Version 2009, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-3289914935, revision: github-musescore-musescore-abc123456

**Comment by Tantacrul** [comment]
@oktophonie

**Comment by MarcSabatella** [comment]
This is correct, according to how this was designed and has always worked in the past.  System breaks in non-mmrest mode *shouldn't* normally affect mmrests.  To break an mmrest, normally you'd add a double bar, rehearsal mark, or other marking that breaks it automatically.  Or use Measure Properties to break it if you need to do so for no apparent reason.  And then you can *also*, if you want, add a system break.  But breaking the rest should not be contingent on the presence of system breaks.

**Comment by Gai-Luron** [comment]
In the end, you are right and your arguments are valid. It's annoying because sometimes it slows down the workflow. It was just a food for thought.

**Comment by MarcSabatella** [comment]
@Gai-Luron feel free to ask for advice on this on the support forum.  Normally, you *shouldn't* break multimeasure rests for no reason, so adding a double bar or other marking would be exactly the right thing to do anyhow.  If you post to the forum and attach your score and describe what you're trying to do, we can advise on the best way to achieve it.
